GCRCatalogs/parquet.py

```diff
@@ -3,6 +3,20 @@
 
 __all__ = ['ParquetFileWrapper']
 
+
+def _retrive_data_from_arrow_table(table, as_dict=False):
```

Comment:
"retrive" is not an English word.  I think you meant "retrieve" so method should be named

_retrieve_data_from_arrow_table

Reply:
Thanks for catching this! Fixed now.